omod/src/main/java/org/openmrs/module/appointments/web/service/impl/DailyRecurringAppointmentsGenerationService.java

```diff
@@ -61,7 +60,11 @@ public Date getEndDate(int period, Integer frequency, Date endDate) {
     public List<Appointment> addAppointments(AppointmentRecurringPattern appointmentRecurringPattern,
                                              RecurringAppointmentRequest recurringAppointmentRequest){
         this.recurringAppointmentRequest = recurringAppointmentRequest;
-        List<Appointment> appointments = appointmentRecurringPattern.getAppointments().stream().collect(Collectors.toList());
+        List<Appointment> activeAppointments = new ArrayList<>(appointmentRecurringPattern.getActiveAppointments());
```

Comment:
Why are we changing this as Part of this PR?

Reply:
@bsneha90 As part of raising events for recurring appointments, we had to consider the voided appointments in the bridge table. The changes are specific to that !